frontend/src/components/Header/index.tsx

```diff
@@ -0,0 +1,42 @@
+import Container from "../util/Container";
+import { tertiaryColor } from "../../utils/styleVariables";
+import { NavLink, Link } from "react-router-dom";
+import ContainerLayout from "../util/ContainerLayout";
+import HeaderContainer from "./HeaderContainer";
+import Logo from "./Logo";
+import NavElement from "./NavElement";
+import Nav from './Nav';
+import { FC } from "react";
+
+const Header:FC = ({children}) => {
+  return (
+  <>
+    <ContainerLayout background={tertiaryColor}>
+      <Container>
+        <HeaderContainer>
+
+          <Logo>
+            <Link to='/'>Project Management</Link>
```

Comment:
`<Logo />` renders `<a />` element, also `<Link />` renders `<a  />` element. 
By the standard of w3c you can not have `a` tag inside of another `a` tag

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element

Reply:
Okk